test/unit/view.js

```diff
@@ -151,101 +247,85 @@ describe('View class', () => {
+        appendChildStub.restore();
       });
     });
 
     describe('destroy()', () => {
       it('removes node from parent', () => {
-        const component = new Component({
-          id: 1234,
-        }, {browserFeatures: {}});
-        const view = new View(component);
         view.node = {
           parentNode: {
             removeChild: sinon.spy(),
           },
         };
         view.destroy();
         assert.calledOnce(view.node.parentNode.removeChild);
+        assert.calledWith(view.node.parentNode.removeChild, view.node);
       });
     });
 
     describe('renderChild()', () => {
-      it('calls updateNode with node and html', () => {
-        const component = new Component({
-          id: 1234,
-          options: {
-            product: {
-              viewData: {
-                title: 'lol',
-              },
-            },
-          },
-        }, {browserFeatures: {}});
-        component.typeKey = 'product';
-        const view = new View(component);
+      it('updates node with new node and html created from class name and template params', () => {
+        const html = '<h1>test</h1>';
+        const node = document.createElement('div');
         const template = {
-          render: sinon.stub().returns('<h1>BUY MY BUTTONS lol</h1>')
+          render: sinon.stub().returns(html),
         };
-        view.updateNode = sinon.spy();
-        view.wrapper = document.createElement('div');
-        const childNode = document.createElement('div');
-        childNode.className = 'title';
-        view.wrapper.appendChild(childNode);
+        view.wrapper = {querySelector: sinon.stub().returns(node)};
+        const updateNodeStub = sinon.stub(view, 'updateNode');
+
+        view.renderChild('class1 class2', template);
 
-        view.renderChild('title', template);
-        assert.calledWith(view.updateNode, childNode, '<h1>BUY MY BUTTONS lol</h1>');
-        assert.calledWith(template.render, sinon.match.has('data'));
+        assert.calledOnce(view.wrapper.querySelector);
+        assert.calledWith(view.wrapper.querySelector, '.class1.class2');
+
+        assert.calledOnce(template.render);
+        assert.deepEqual(template.render.getCall(0).args[0], {data: view.component.viewData});
+
+        assert.calledOnce(updateNodeStub);
+        assert.calledWith(updateNodeStub, node, html);
+
+        updateNodeStub.restore();
       });
     });
 
     describe('updateNode()', () => {
-      const component = new Component({
-        id: 1234,
-      }, {browserFeatures: {}});
-      const view = new View(component);
-
       it('updates contents of node', () => {
         const div = document.createElement('div');
-        div.innerHTML = '<h1>OLD TEXT</h1>';
-        const html = '<h1>SO FRESH</h1>';
+        div.innerHTML = '<h1>old</h1>';
+        const html = '<h1>new</h1>';
         view.updateNode(div, `<div>${html}</div>`);
         assert.equal(div.innerHTML, html);
```

Comment:
This test is a bit inconsistent with others, since we're testing that morphdom works correctly here, not that we're calling it with the expected things.